ServerCore/Pages/Teams/AutoTeam.razor.cs

```diff
@@ -89,9 +110,19 @@ where teamGroup.Count() < maxAutoTeamSize
                     AutoTeamType = playerType,
                     Name = name,
                     PrimaryContactEmail = captain.Email,
+                    IsRemoteTeam = isRemote,
                 };
 
-                await TeamHelper.CreateTeamAsync(_context, team, Event, LoggedInUserId);
+                try
+                {
+                    await TeamHelper.CreateTeamAsync(_context, team, Event, LoggedInUserId);
+                }
+                catch
+                {
+                    // Most exceptions from here are because the team type isn't available

```

Comment:
Do you need to tell the user this?

Reply:
It will show "All teams with that choice are full. Please try another or apply to an existing team". Is that good enough?